.github/ISSUE_TEMPLATE/New_release.md

```diff
@@ -14,36 +14,34 @@ This issue is to provide visibility on the progress of the release process of Gu
 ## Resources
 
 -   📖 Read: [Gutenberg Release Process](https://developer.wordpress.org/block-editor/contributors/code/release/)
--   📖 Read: [Leading a Gutenberg Plugin Release](https://codep2.wordpress.com/2021/07/22/leading-a-gutenberg-plugin-release/)
 -   📽 Watch: [Gutenberg Plugin: New Release Workflow](https://www.youtube.com/watch?v=TnSgJd3zpJY)
 -   📽 Watch: [Creating the Gutenberg plugin v12.0 Release Candidate](https://www.youtube.com/watch?v=FLkLHKecxWg)
 -   📽 Watch: [Gutenberg plugin v12.0.0 Release Party!](https://www.youtube.com/watch?v=4SDtpVPDsLc)
 
 ## Checklist
 
-### RC Day - Wednesday, March 15
+### RC Day - {Weekday, Month, Date}
 
 -   [ ] _Optional:_ Attend `#core-editor` meeting (14:00UTC)
 -   [ ] Post a message in `#core-editor` channel to let folks know you are starting the RC release process
 -   [ ] Organize and Label PRs on the relevant milestone
```

Comment:
Should this be done before RC day? Or at least before posting in #core-editor that you're about to start the release process?

Reply:
I'm not sure I understand. These are the steps that need to be taken during the RC release process.